src/main/java/blue/UI.java

```diff
@@ -0,0 +1,96 @@
+package blue;
+
+import java.util.ArrayList;
+
+import blue.task.Task;
+
+
+
+/**
+ * The {@code UI} class handles interactions with the user by displaying messages and task information.
+ * It provides methods to greet the user, say farewell, and display task-related updates.
+ */
+public class UI {
```

Comment:
Based on standard naming convention, abbreviation (UI) shouldn't be uppercase when used as part of a name. Should it be Ui instead for the name of this class?